CppCoreGuidelines.md

```diff
@@ -2258,7 +2258,7 @@ See [GOTW #100](https://herbsutter.com/gotw/_100/) and [cppreference](http://en.
 ##### Reason
 
 To keep code simple and safe.
-Sometimes, ugly, unsafe, or error-prone techniques are necessary for logical or performance reasons.
```

Comment:
Changing "ugly" to "verbose" (in most places) kind of changes the meaning... we view "ugly code" as accepted technical jargon, but we also have tried to think of a better alternative that doesn't change the meaning here, if you can suggest a different edit we could consider that otherwise we'd prefer to leave this as is.

Reply:
reverted